pom.xml

```diff
@@ -1893,6 +1891,9 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-javadoc-plugin</artifactId>
+            <configuration>
+              <failOnWarnings>false</failOnWarnings>
+            </configuration>
```

Comment:
This causes build to [succeed](https://github.com/Daniilchik/ozone/actions/runs/11092674433/job/30818473876#step:8:19567) despite [javadoc warnings](https://github.com/Daniilchik/ozone/actions/runs/11092674433/job/30818473876#step:8:4848).

```suggestion
```

Reply:
I've set this one in true and compile(8, macos-12) throws error [error](https://github.com/Daniilchik/ozone/actions/runs/11093722021/job/30820762982#step:8:6706) even though I've set `-Dmaven.javadoc.failOnWarnings=false`

Reply:
Fixed it by removing `<failOnWarnings></failOnWarnings>` from `pom.xml` so we can use
`-Dmaven.javadoc.failOnWarnings` in compile check